CMakeLists.txt

```diff
@@ -519,8 +519,8 @@ hpx_option(HPX_WITH_THREAD_STACK_MMAP BOOL
   CATEGORY "Thread Manager" ADVANCED)
 
 hpx_option(HPX_WITH_THREAD_MANAGER_IDLE_BACKOFF BOOL
-  "HPX scheduler threads do exponential backoff on idle queues (default: ON)"
-  ON
+  "HPX scheduler threads do exponential backoff on idle queues (default: OFF)"
```

Comment:
Side note: the `wait_count_` variable would also be a candidate for "unnecessary" sharing (i.e. it's shared across all threads, whereas there could easily be one per thread). I think turning this off is a really good default though.

In the worst case the idle backoff makes things slower because of additional latency from waking threads up, but in the best case it might even help reduce contention on thread queues.